client/gutenberg/extensions/map-block/config.js

```diff
@@ -0,0 +1,338 @@
+/**
+ * Wordpress dependencies
+ */
+
+import { __ } from '@wordpress/i18n';
+
+/**
+ * External dependencies
+ */
+
+ /**
+ * Internal dependencies
+ */
+
+export const CONFIG = {
+	name: 'atavist/maps',
+	title: __( 'Map' ),
```

Comment:
If this will be a Jetpack block, does it make sense to have the `jetpack` textdomain? What are our plans on handling and serving translations for these blocks?

Reply:
Not sure I understand this one - could you give me some context?

Reply:
Well, in order for `wp.i18n` to know where to look for translations for a block, it needs a textdomain. In `wp.i18n.__()`, this is the second argument of the function. In this particular call, we seem to be lacking a textdomain.

If you have a look ot some of the other Jetpack blocks, we use the `jetpack` textdomain, which will later allow us to use Jetpack for indexing translations, then managing and serving them.

Let me know if that makes sense.

Reply:
Yes, thank you for the explanation. I believe I got them all in https://github.com/Automattic/wp-calypso/commit/0241560c0f84181a9c2f4fedfe30f90ca084d0f4. Could you take a look and verify I did this correctly?

Reply:
All the `__()` cases look good to me 👍 The only case that seem to have been missed is the `_n()` one - https://github.com/Automattic/wp-calypso/pull/27836/files#diff-7e7cc0bd139c6be3b8dcca033daea924R169 - it needs the textdomain as a fourth argument.

Reply:
Caught the `_n()` case in https://github.com/Automattic/wp-calypso/pull/27836/commits/bf7f6a08e4fe3e5f4314c556bf6458c62eaa6a1e.